pkg/kubectl/cmd/attach.go

```diff
@@ -45,8 +45,7 @@ var (
 		# Get output from ruby-container from pod 123456-7890
 		kubectl attach 123456-7890 -c ruby-container
 
-		# Switch to raw terminal mode, sends stdin to 'bash' in ruby-container from pod 123456-7890
-		# and sends stdout/stderr from 'bash' back to the client
+		# Switch to raw terminal mode, sends stdin to 'bash' in ruby-container from pod 123456-7890 and sends stdout/stderr from 'bash' back to the client
```

Comment:
How does the line break look like when you display this help in a terminal with ~120 columns?

Reply:
It looks terrible. But this is a problem any time the line length is longer than the column width of the terminal. If we couldn't agree on 80 how are we to agree on 120? The design reasoning behind the change was instead of the column width being hard coded the line wrapping should be handled by the tool (cobra [templates]). It currently is not. In general our help output is a bit clutter IMO but it is a hard problem to solve because we seem to provide long descriptions (which seems good). For contrast try `docker help build`. I posted else where that perhaps output format should be a problem solved above us i.e since every user has different likes/needs the user knows best how they want the output. One solution I found was using a colorization package, I'm sure there are others. Reducing information for ease of formatting does not seem like a good idea.

Another possible idea is `--column-width` flag that does as the name suggests.